api/model/src/test/java/org/projectnessie/model/TestNessieConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.projectnessie.model.NessieConfiguration.getConfigWithPredefinedVersions;
+
+import org.junit.jupiter.api.Test;
+
+public class TestNessieConfiguration {
+  @SuppressWarnings({"Convert2MethodRef", "ResultOfMethodCallIgnored"})
+  @Test
+  public void checkPredefinedVersions() {
+    // Only verify that loading the NessieConfiguration.json works and does not throw.
+    assertThatCode(() -> getConfigWithPredefinedVersions()).doesNotThrowAnyException();
+    assertThat(getConfigWithPredefinedVersions()).isNotNull();
```

Comment:
nit: the check on line 29 is implied in line 30.

Reply:
Not really - not-null

Reply:
If it throws, line 30 alone will fail just as well :shrug: